entity-framework/core/what-is-new/ef-core-5.0/breaking-changes.md

```diff
@@ -16,6 +16,10 @@ The following API and behavior changes have the potential to break existing appl
 |:--------------------------------------------------------------------------------------------------------------------------------------|------------|
 | [Required on the navigation from principal to dependent has different semantics](#required-dependent)                                 | Medium     |
 | [Removed HasGeometricDimension method from SQLite NTS extension](#geometric-sqlite)                                                   | Low        |
+| [Cosmos: Partition key is now added to the primary key](#cosmos-partition-key)                                                        | Low        |
+| [Cosmos: `id` property renamed to `__id`](#cosmos-id)                                                                                 | Low        |
+| [Cosmos: Cosmos: byte[] is now stored as a base64 string instead of a number array](#cosmos-byte)                                     | Low        |
```

Comment:
```suggestion
| [Cosmos: byte[] is now stored as a base64 string instead of a number array](#cosmos-byte)                                     | Low        |
```